apis/telemetry/v1alpha1/metricpipeline_types.go

```diff
@@ -63,36 +63,87 @@ type MetricPipelineSpec struct {
 
 // MetricPipelineInput defines the input configuration section.
 type MetricPipelineInput struct {
-	// Configures application related scraping.
-	Application MetricPipelineApplicationInput `json:"application,omitempty"`
-}
-
-// MetricPipelineApplicationInput defines the application input configuration section.
-type MetricPipelineApplicationInput struct {
 	// Configures Prometheus scraping.
 	Prometheus MetricPipelinePrometheusInput `json:"prometheus,omitempty"`
 	// Configures runtime scraping.
 	Runtime MetricPipelineContainerRuntimeInput `json:"runtime,omitempty"`
 	// Configures istio-proxy metrics scraping.
 	Istio MetricPipelineIstioInput `json:"istio,omitempty"`
+	// Configures the collection of push-based metrics which are using the OpenTelemetry protocol.
```

Comment:
```suggestion
	// Configures the collection of push-based metrics that are using the OpenTelemetry protocol.
```

Do you really mean the **are using it** *right now*? If they use it _in general_, I suggest:

```suggestion
	// Configures the collection of push-based metrics that use the OpenTelemetry protocol.
```

Reply:
They use the OpenTelemetry protocol in general , so I applied your 2nd suggestion  😄